#goal definition
char[] position
---
#result definition
bool isOK
---
#feedback
int32[] sequence
